app/javascript/components/pages/SpotIndex.js

```diff
@@ -1,9 +1,55 @@
 import React, { Component } from 'react'
+import { Card, CardImg, CardBody, CardTitle, CardSubtitle, Button, Col, Row } from 'reactstrap'
 
 class SpotIndex extends Component {
+    constructor(props) {
+        super(props)
+    }
   render() {
     return (
-      <div>SpotIndex</div>
+      <>
+        {this.props.logged_in && 
```

Comment:
Logic looks great in here, but you can probably consolidate this down so you only need to conditionally render this.props.logged_in once.